integration-cli/docker_experimental_network_test.go

```diff
@@ -1,4 +1,4 @@
-// +build experimental
+// +build !windows
```

Comment:
Were we just never running tests on `windows` with `experimental` enabled previously?